index.html

```diff
@@ -8,7 +8,9 @@
     <title>Rock-Paper-Scissors-Game</title>
 
     <link rel="stylesheet" href="./Styles/index.css">
-    <script src="./Scripts/index.js"></script>
+
+    <!-- "defer" allows the JS file to run once the HTML file is finished being read -->
+    <script src="./Scripts/index.js" defer></script>
```

Comment:
`defer` attribute was added to `<script>` so the JavaScript code parses after the HTML file has finished loading